Dialogs/PreferencesDialog.ui

```diff
@@ -147,24 +153,44 @@
            <property name="bottomMargin">
             <number>0</number>
            </property>
-           <item>
-            <widget class="QTabWidget" name="tabWidget">
-             <property name="autoFillBackground">
-              <bool>false</bool>
+           <item row="0" column="1">

```

Comment:
I assume this was generated by something. It's consistent, but it's not very pretty.

Reply:
The designer yeah, that thing is difficult to fight too, I've tried fighting the designer on other things too. It loves making a mess out of what you change in the UI file.